form.pug

```diff
@@ -1,15 +1,15 @@
-form.contact-form
-	.contact-form__items 
+form.contactForm
+	.contactForm__items 
```

Comment:
Why are you changing the BEM syntax to `contactForm` (camelCase approach) instead of `contact-form`

Reply:
because I thought it would be confused with the modifier syntax.  I will correct it